### PR TITLE
Bump dcos-metrics sha

### PR DESCRIPTION
**Pull request by philipnrmn**
## High-level description

What features does this change enable? What bugs does this change fix?
This PR brings the version of dcos-metrics in DC/OS up to date with the current dcos-metrics master. The two significant changes here both add new metrics to the </containers/container-id> endpoint in dcos-metrics. 

Note that there is a lot of activity in the changelog linked below, all related to plugins and documentation. 

## Corresponding DC/OS tickets (obligatory)

These DC/OS JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:

 - [DCOS_OSS-1607](https://jira.mesosphere.com/browse/DCOS_OSS-1607) Add container resource utilisation metrics
 - [DCOS_OSS-1489](https://jira.mesosphere.com/browse/DCOS_OSS-1489) Support cgroup blkio statistics in dcos-metrics.

 - [DCOS-19920](https://jira.mesosphere.com/browse/DCOS-19920) Update vendored dcos-go in dcos-metrics
 - [DCOS-19984](https://jira.mesosphere.com/browse/DCOS-19984) Mesos module for dcos-metrics fails during dcos build

## Checklist for all PRs

  - [x] Included a test which will fail if code is reverted but test is not. If there is no test please explain here:
  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [x] Change log from the last version integrated (this should be a link to commits for easy verification and review): [change log](https://github.com/dcos/dcos-metrics/compare/5bbbb585d8e600073f6dc8946a3c0b4476ab7fff...d08d02d048b4a2997b5f2a2836ee08c4197482c4)
  - [x] Test Results: [link to CI job test results for component](https://jenkins.mesosphere.com/service/jenkins/job/public-dcos-cluster-ops/job/dcos-metrics/job/dcos-metrics-master/153/testReport/)
  - [x] Code Coverage (if available): [link to code coverage report](https://jenkins.mesosphere.com/service/jenkins/job/public-dcos-cluster-ops/job/dcos-metrics/job/dcos-metrics-master/153/cobertura/)

### CODE REVIEW

**Comment by philipnrmn** [comment]
@mesosphere-mergebot bump-ee

**Comment by philipnrmn** [comment]
@mesosphere-mergebot label ready for review

**Comment by philipnrmn** [comment]
@collinvandyck and @mnaboka , would you kindly take a look at this PR?

**Comment by orsenthil** [comment]
@philipnrmn  - `teamcity/dcos/build/dcos` failure should be investigated. It is a primary task that builds the DCOS Image.

**Comment by philipnrmn** [comment]
Hi @orsenthil, it looks like there's an error in the mesos module build. I'll see if I can get some help with this.

**Comment by philipnrmn** [comment]
@mesosphere-mergebot label work in progress

**Comment by philipnrmn** [comment]
@mesosphere-mergebot label ready for review

**Comment by philipnrmn** [comment]
@mesosphere-mergebot bump-ee

**Comment by philipnrmn** [comment]
Rebased and squashed

**Comment by philipnrmn** [comment]
@mesosphere-mergebot bump-ee

**Comment by philipnrmn** [comment]
@mesosphere-mergebot bump-ee

**Comment by philipnrmn** [comment]
@mesosphere-mergebot bump-ee

**Comment by philipnrmn** [comment]
(I closed mesosphere/dcos-enterprise#1880 because it didn't pick my rebase up)

**Comment by philipnrmn** [comment]
@mnaboka, @collinvandyck - would you mind re-approving this PR? I had to rebase.

File: packages/dcos-integration-test/extra/test_metrics.py [hunk 1]
```diff
@@ -139,7 +139,8 @@ def wait_for_container_propogation():
                 cid_registry = []
                 for dp in container_response.json()['datapoints']:
                     assert 'tags' in dp, 'got {}'.format(dp)
-                    assert len(dp['tags']) == 5, 'got {}'.format(
+                    # blkio stats have 'device' tags as well
+                    assert len(dp['tags']) >= 5, 'got {}'.format(
```
Comment (collinvandyck) [inline packages/dcos-integration-test/extra/test_metrics.py]: If we've added blkio tags/metrics, do we want to test that `len(dp['tags']) > 5` instead?

Comment (philipnrmn) [reply packages/dcos-integration-test/extra/test_metrics.py]: Not all the container stats are blkio stats - the existing (mem, cpu, gpu, network I/O) stats don't have blkio source device tags, so they still have five.

Comment (collinvandyck) [reply packages/dcos-integration-test/extra/test_metrics.py]: Aha, I get it now -- thanks.

**Comment by philipnrmn** [comment]
@mesosphere-mergebot bump-ee

**Comment by philipnrmn** [comment]
Rebased again, killed the ee branch again.

**Comment by philipnrmn** [comment]
@mesosphere-mergebot bump-ee

**Comment by philipnrmn** [comment]
@mesosphere-mergebot label ship it

**Pull request merged**